src/transaction/log_applier.c

```diff
@@ -2962,7 +2962,7 @@ la_add_apply_list (int tranid)
  *   log_pageid, log_offset, and log_pgptr are set as a side effect.
  */
 static void
-la_log_copy_fromlog (char *rec_type, char *area, int length, LOG_PAGEID log_pageid, PGLENGTH log_offset,
+la_log_copy_fromlog (char *rec_type, char *area, int &length, LOG_PAGEID log_pageid, PGLENGTH log_offset,
```

Comment:
I'm not sure if it's better to change `int length` to `int &length`.
Wouldn't it be better to specify it as an (in/out) parameter at function comment and receive it as a pointer type ('int *length) rather than adding a reference type to the c style code?

Reply:
I will. I had the same thought, but I didn't do it because I had to modify all the callers. I wanted to minimize modifications.

Reply:
I applied your feedback. Re-review plz.